lib/driver.rb

```diff
@@ -42,6 +43,34 @@ def add_trip(trip)
       end
 
       @trips << trip
+    end # add_trip
+
+    def total_revenue
+      tax = 1.65
+      driver_pay = 0.8
+      gross_pay = 0
+
+      trips.each do |trip|
+        gross_pay += trip.cost - tax
+      end
+      total_revenue = gross_pay * driver_pay
+      return total_revenue
     end
-  end
-end
+
+    # calulate drivers average revenue per hour spend driving
+    def average_revenue
+      # input: total_revenue, total_time_spent(hours),
+      # output: average_revenue = total_revenue / total_time_spent
+      average_revenue = 0
+      total_time = Trip.total_time(trips)/60/60.round(2)
+      average = self.total_revenue / total_time
+      average_revenue += average
+      return average_revenue.round(2)
```

Comment:
If you're rounding the final value (line 68), you shouldn't need to also round an intermediate value (line 65).

Also, I believe the `.` will bind tighter than the `/`, so the `round(2)` will be applied to the last `60`. If you wanted to round the whole thing you would have to use parentheses: `(Trip.total_time(trips)/60/60).round(2)`